### PR TITLE
Added browser compatibility table for input type search

### PR DESCRIPTION
**Pull request by varun07**
References:

1. [Can I Use?](https://caniuse.com/#feat=input-search)
2. [https://www.wufoo.com/html5/](https://www.wufoo.com/html5/)

### CODE REVIEW

File: html/elements/input/search.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "5"
+              },
+              "safari_ios": {
+                "version_added": true
+              }
```
Comment (ddbeck) [inline html/elements/input/search.json]: Could we add `"webview_android"`, too? I think we can safely use the `true` value.

File: html/elements/input/search.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
```
Comment (ddbeck) [inline html/elements/input/search.json]: This trailing white space can go too

File: html/elements/input/search.json (hunk 1 above)
Comment (varun07) [reply html/elements/input/search.json]: I think we can use that

File: html/elements/input/search.json (hunk 2 above)
Comment (varun07) [reply html/elements/input/search.json]: Okay 👍

**Comment by varun07** [comment]
Have added the changes as asked.

**Comment by ddbeck** [comment]
Excellent, thank you so much! 🎉

**Pull request merged**